Source/AwsCommonRuntimeKit/io/TLSContextOptions.swift

```diff
@@ -33,6 +45,32 @@ public class TLSContextOptions: CStruct {
         }
     }
 
+    init(certificateData cert_data: String,
+         privateKeyData private_key_data: String) throws {
+        self.rawValue = allocator.allocate(capacity: 1)
+        if( cert_data.withByteCursorPointer { certificateByteCursor in
```

Comment:
- guard let is better for this case because it ensures that we don't fall through the error branch.
- You can avoid nesting withByteCursorPointer by using the helper function like https://github.com/awslabs/aws-crt-swift/blob/main/Source/AwsCommonRuntimeKit/http/HTTPProxyOptions.swift#L37

Reply:
Updated to use an extra help function to pass the bytecursor.